README.md

```diff
@@ -13,21 +13,23 @@ This repository is effectively frozen. We recommend that developers of new token
 You can install from npm with `npm install eth-contract-metadata` and use it in your code like this:
 
 ```javascript
-const contractMap = require('eth-contract-metadata')
-const toChecksumAddress = require('ethereumjs-util').toChecksumAddress
+import contractMap from 'eth-contract-metadata'
+import ethJSUtil from 'ethereumjs-util'
+const { toChecksumAddress } = ethJSUtil
```

Comment:
```suggestion
import { toChecksumAddress } from 'ethereumjs-util'
```

Reply:
@whymarrh in order for this to work we'd need to make some additional changes:

```js
$ node index.js 
file:///home/ricky/projects/eth-contract-metadata-example/index.js:5
import { toChecksumAddress } from 'ethereumjs-util'
         ^^^^^^^^^^^^^^^^^
SyntaxError: The requested module 'ethereumjs-util' is expected to be of type CommonJS, which does not support named exports. CommonJS modules can be imported by importing the default export.
For example:
import pkg from 'ethereumjs-util';
const { toChecksumAddress } = pkg;
```